_posts/2017-01-04-OSS-FAQ.md

```diff
@@ -0,0 +1,69 @@
+
+### What is Open Source?
+
+Open Source code is code that can be freely examined, used, adapted, and shared by all through a license that sets forth these principles.  The only potential limitation that an Open Source license is likely to impose is that future copies of the code (whether in adapted or un-adapted form) be themselves licensed in a manner consistent with the original license.  At Artsy, we are committed to making our engineering work [Open Source by default](http://code.dblock.org/2015/02/09/becoming-open-source-by-default.html).  A list of our Open Source projects can be [found here](http://artsy.github.io/open-source/).
+
+<!-- more -->
+
+### What is Open Source “by default”?
+
+Open Source “by default” means that new projects are started or eventually become Open Source, unless keeping them proprietary represents a significant competitive advantage. 
+
+### What are the advantages to Open Source code?
+
+Keeping our engineering work Open Source by default allows us to be a thought leader,  attracts and retains the best talent, and also is a great way to produce high quality work -- in other words, it helps to drive and sustain our Quality Worthy of Art, Openness, and People are Paramount values.  However, not everything is or should be Open Source. There [are many great reasons](https://www.objc.io/issues/22-scale/artsy/) to be open, but it’s not a catch all. 
+
+### How does Open Source interact with Copyright?
+
+Very generally speaking, copyright is the body of law that governs how Artsy upholds its ownership rights over code that an Artsy engineer creates as part of their duties as an Artsy employee, and Open Source describes the way in which Artsy shares this code with the rest of the world.  Put in another way, Artsy engineers develop code as "work for hire" to Artsy, and Artsy shares this code with specific usage and distribution terms through a license attached to it.
+
+### What's Inbound Open Source code vs. Outbound Open Source code?
+
+Inbound Open Source code refers to the use of third party code under an Open Source license for Artsy's products.  Outbound Open Source code refers to the publication of Artsy code.  
+
+When we publish Outbound Open Source code, as mentioned before, Artsy has a right to do so because we have the copyright to this code.  We give third parties the right to use the code usually through [this  type](https://github.com/artsy/eigen/blob/master/LICENSE) of broad license ([MIT](https://tldrlegal.com/license/mit-license)).  (For more on code licenses, see [Orta’s talk on code licenses](http://artsy.github.io/blog/2015/12/10/License-and-You/.)
```

Comment:
The period needs to be outside of the parenthesis, it's part of the sentence, not the link. The link here is broken.